lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -142,6 +297,20 @@ class ProgramLoweringState {
   DenseMap<Operation *, ComponentLoweringState> compStates;
 };
 
+/// Creates register assignment operations within the provided groupOp.
+static void buildAssignmentsForRegisterWrite(ComponentLoweringState &state,
+                                             PatternRewriter &rewriter,
+                                             calyx::GroupOp groupOp,
+                                             calyx::RegisterOp &reg, Value v) {
```

Comment:
Better name than `v` please. It seems this is used as the input value for the RegisterOp?